odoo/http.py

```diff
@@ -1232,6 +1232,7 @@ def render(self):
         """
         env = request.env(user=self.uid or request.uid or odoo.SUPERUSER_ID)
         self.qcontext['request'] = request
+        self.qcontext['get_modules_order'] = lambda: json.dumps(module_boot())
```

Comment:
this should use `json_scriptsafe.dumps`, not normal `json.dumps`, like all serialized json data that gets inserted into a `<script>` block